shared/templates/static/ansible/file_permissions_etc_group.yml

```diff
@@ -0,0 +1,13 @@
+# platform = multi_platform_rhel
```

Comment:
I just copied the platforms from [here](https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/templates/static/bash/file_permissions_etc_group.sh)

I'm planning to improve somehow this platform attribute, it's is against DRY as we already have the platform in the path. But I'm not sure how to do it yet.